Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -251,6 +257,14 @@ public class Snapshotter {
                                 bearing: bearing?.NSNumber,
                                 pitch: pitch?.NSNumber))
     }
+
+    // MARK: - Telemetry
+    internal func sendTurnstileEvent() {
+        let accessToken = ResourceOptionsManager.default.defaultAccessToken()
+        let eventsManager = EventsManager(accessToken: accessToken)
+
+        eventsManager.telemetry.turnstile()
```

Comment:
Also, should we send a turnstile with each snapshot or just once per Snapshotter (e.g. after the first successful snapshot for a given Snapshotter)?